build.psm1

```diff
@@ -468,6 +468,16 @@ cmd.exe /C cd /d "$location" "&" "$($vcVarsPath)\vcvarsall.bat" "$NativeHostArch
         }
         New-Item -Path $refDestFolder -ItemType Directory -Force -ErrorAction Stop > $null
         Copy-Item -Path $refAssemblies -Destination $refDestFolder -Force -ErrorAction Stop
+
+        # create the symlinks needed on Linux
+        if ($IsRedHatFamily) {
+            if ( ! (test-path "$publishPath/libssl.so.1.0.0")) {
+                $null = New-Item -Force -ItemType SymbolicLink -Target "/lib64/libssl.so.10" -Path "$publishPath/libssl.so.1.0.0"
```

Comment:
-ErrorAction Stop for both New-Item calls. That is the pattern used in the function.